tests/Traces/README

```diff
@@ -1,13 +1,32 @@
-Traces from Zeek:
-    - dns53.pcap
-    - http-post.pcap (post.trace) 
-    - png.pcap (pipelined-requests.trace)
-    - ftp-pe.pcap (pe/pe.trace, pruned a single file transfer)
-    
-Traces from https://wiki.wireshark.org/SampleCaptures:
-    - dhcp.pcap
-    - tftp_rrq.pcap
-    - tftp_wrq.pcap
+The test suite comes with a set of traces collected from a variety of
+places that we document below. While these traces are all coming from
+public sources, please note that they may carry their own licenses.
+We collect them here for convenience only.
 
+DHCP
+- [dhcp.pcap](https://gitlab.com/wireshark/wireshark/-/wikis/uploads/__moin_import__/attachments/SampleCaptures/dhcp.pcap)
 
+DNS
+- [dns53.pcap](https://github.com/zeek/zeek/blob/master/testing/btest/Traces/dns53.pcap)
 
+HTTP
+- [http-post.pcap](https://github.com/zeek/zeek/blob/master/testing/btest/Traces/http/post.trace)
+
+OpenVPN
+- [openvpn-tcp-tls-auth.pcap](https://wiki.wireshark.org/SampleCaptures?action=AttachFile&do=get&target=OpenVPN_TCP_tls-auth.pcapng)
```

Comment:
This link seems to be deprecated and by banner redirects to https://wiki.wireshark.org/SampleCaptures, but the file is listed on neither of the overview pages. I am not sure what that means for us.

Reply:
Yeah, I didn't find the trace at the new location. That old link still leads directly to download for me, that's why I just left it in as is.